GameData/RealismOverhaul/Waterfall_Configs/NearFutureTechnologies/NearFutureLV_engines.cfg

```diff
@@ -0,0 +1,106 @@
+//	=================================================================================
+//	Near Future Launch Vehicles Engines
+//	=================================================================================
+//	Nertea did a pretty good job with these, just rescale the plumes and remove the stock smoke
+@PART[nflv-engine-*]:BEFORE[ROWaterfall]:NEEDS[Waterfall]
+{
+	//kill the stock smoke plumes
+	@EFFECTS
+	{
+		@running_closed
+		{
+			!PREFAB_PARTICLE {}
+		}
+	}
+
+	//just rescale the plumes
+	@MODULE[ModuleWaterfallFX]
+	{
+		@TEMPLATE
+		{
+			@scale[*] *= #$/rescaleFactor$
+		}
+	}
+}
+
+// RD-701 and RD-704 get their plumes adjusted so to support bimodal operation
+// WIP, needs fixed
+@PART[nflv-engine-rd701-1|nflv-engine-rd704-1]:BEFORE[ROWaterfall]:NEEDS[Waterfall]
+{
+	//kill the stock waterfall FX
+	!MODULE[ModuleWaterfallFX] {}
```

Comment:
ROWaterfall does this for you:

https://github.com/KSP-RO/RealismOverhaul/blob/8ca49b1dab4238494834275711ac2846a36759c3/GameData/RealismOverhaul/Waterfall_Configs/_Processor/_early-initialize.cfg#L10-L15